.tool-versions

```diff
@@ -1,2 +1,3 @@
-erlang 26.2.1
-elixir 1.16.0-otp-26
+elixir 1.17.3
+erlang 26.2.5
```

Comment:
we should be OTP 27? I think we should update other repo too

Reply:
Updated to 27.2.1